modules/ossm-installing-multi-primary-multi-network-mesh.adoc

```diff
@@ -16,15 +17,34 @@ You can adapt these instructions for a mesh spanning more than two clusters.
 
 .Prerequisites
 
-* You have installed the {SMProduct} 3 Operator on all of the clusters that comprise the mesh.
+* You have installed the {SMProduct} 3 Operator on all of the clusters that include the mesh.
 
-* You have completed "Creating certificates for a multi-cluster mesh".
+* You have created certificates for the multi-cluster mesh.
 
-* You have completed "Applying certificates to a multi-cluster topology".
+* You have applied certificates to the multi-cluster topology.
 
 * You have created an {istio} Container Network Interface (CNI) resource.
 
-* You have `istioctl` installed on the laptop you can use to run these instructions.
+* You have `istioctl` installed.
+
+[IMPORTANT]
+====
+In on-premise environments, such as those running on bare metal, {ocp-product-title} clusters often do not include a native load-balancer capability. So, a service of type `LoadBalancer`, such as the `istio-eastwestgateway`, will not automatically be assigned an external IP address.
```

Comment:
So, a service of type `LoadBalancer`, such as the `istio-eastwestgateway`, will not automatically be assigned an external IP address. -> A service of type `LoadBalancer`, such as the `istio-eastwestgateway`, will not automatically be assigned an external IP address.